prow/test/integration/test/webhook_server_test.go

```diff
@@ -118,8 +126,151 @@ func TestWebhookServerMutateProwjob(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unable to unmarshal into prowjob %v", err)
 	}
+
 	diff := cmp.Diff(originalProwJob.Spec.DecorationConfig, mutatedProwJob.Spec.DecorationConfig)
 	if diff == "" {
 		t.Errorf("prowjob was not mutated %v", err)
 	}
 }
+
+func TestWebhookServerAvailability(t *testing.T) {
+	
+	clusterContext := getClusterContext()
+	kubeClient, err := NewClients("", clusterContext)
+	if err != nil {
+		t.Fatalf("Failed creating clients for cluster %q: %v", clusterContext, err)
+	}
+	ctx := context.Background()
+	t.Cleanup(func() {
+		var testProwjob v1.ProwJob
+		absDeploymentPath, err := filepath.Abs(deploymentPath)
+		if err != nil {
+			logrus.WithError(err).Info("could not find absolute path")
+		}
+		out, err := exec.Command("kubectl", "--context="+getClusterContext(), "apply", "-f", absDeploymentPath).CombinedOutput()
+		if err != nil {
+			logrus.WithError(err).Infof("could not clean up cluster %s", string(out))
+		}
+		key := types.NamespacedName{
+			Namespace: "default",
+			Name: testServerAvailability,
+		}
+		if err := kubeClient.Get(ctx, key, &testProwjob); err != nil {
+			logrus.WithError(err).Infof("prowjob not found")
+		} else {
+			out, err := exec.Command("kubectl", "--context="+getClusterContext(), "delete", "prowjob", testServerAvailability).CombinedOutput()
+			if err != nil {
+				logrus.WithError(err).Infof("could not clean up cluster %s", string(out))
+			}	
+		}
+		
+	})
+	
+	key := types.NamespacedName{
+		Namespace: "default",
+		Name:      "webhook-server",
+	}
+	var found bool
+	var webhookDeployment appsv1.Deployment
+	err = kubeClient.Get(ctx, key, &webhookDeployment)
+	if err != nil {
+		t.Fatalf("could not list deployment %v", err)
+	}
+	found = true
```

Comment:
if the test hasn't failed above then found has to be true, so there is no need to define a `found` var